File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.java
class Solution {
    public int minSteps(String s, String t) {
        int res = 0;
        int[] sCount = new int[26];
        int[] tCount = new int[26];
        
        for(char ch : s.toCharArray()){
            sCount[ch - 'a']++;
        }
        for(char ch : t.toCharArray()){
            tCount[ch - 'a']++;
        }
        
        for(int i = 0; i < 26; i++){
            res += Math.abs(sCount[i] - tCount[i]); 
        }
        return res;
    }
}

File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        vector<int> freqMap(26,0);
		
        for(auto ch: s)
			freqMap[ch-'a']++;
        for(auto ch: t)
			freqMap[ch-'a']--;
			
        int ans=0;
        for(int i=0;i<26;i++)
			ans+=abs(0-freqMap[i]);
			
        return ans;
    }
};